Packs/RunZero/Integrations/RunZero/RunZero.py

```diff
@@ -0,0 +1,525 @@
+    message['TCP'] = raw.get('service_count_tcp', 0)
+    message['UDP'] = raw.get('service_count_udp', 0)
+    message['ICMP'] = raw.get('service_count_icmp', 0)
+    message['ARP'] = raw.get('service_count_arp', 0)
+    message['SW'] = raw.get('software_count', 0)
+    message['Vulns'] = raw.get('vulnerability_count', 0)
+    message['RTT/ms'] = normalize_rtt(raw.get('lowest_rtt', 0))
+    message['Hops'] = raw.get('lowest_ttl', 0)
+    message['Detected'] = raw.get('detected_by', 0)
+    message['First_Seen'] = timestamp_to_datestring(raw.get('first_seen', '') * 1000)
+    message['Last_Seen'] = timestamp_to_datestring(raw.get('last_seen', '') * 1000)
+    message['Explorer'] = raw.get('agent_name', '')
+    message['Hosted_Zone'] = raw.get('hosted_zone_name', '')
+    message['Site'] = raw.get('site_name', '')
+    return [message]
+
+
+def parse_raw_quota_get(raw: dict) -> dict:
+    message = {}
+    message['usage_limit'] = raw.get('created_at', '')
+    message['usage_today'] = raw.get('usage_today', '')
+    message['counter'] = raw.get('counter', '')
+    return message
+
+
+def parse_raw_wireless(raw: dict) -> list:
+    message = {}
+    message['ID'] = raw.get('id', '')
+    message['ESSID'] = raw.get('essid', '')
+    message['BSSID'] = raw.get('bssid', '')
+    message['Vendor'] = raw.get('vendor', '')
+    message['Family'] = raw.get('family', '')
+    message['Type'] = raw.get('type', '')
+    message['Auth'] = raw.get('authentication', '')
+    message['Enc'] = raw.get('encryption', '')
+    message['Sig'] = raw.get('signal', 0)
+    message['Int'] = raw.get('interface', '')
+    message['Additional'] = raw.get('data', {})
+    message['First_seen'] = timestamp_to_datestring(raw.get('created_at', ''))
+    message['Last_seen'] = timestamp_to_datestring(raw.get('last_seen', ''))
+    message['Site'] = raw.get('site_name', '')
+    return [message]
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def asset_search(client: Client, args: dict) -> CommandResults:
+    search_string = ''
+    limit = arg_to_number(args.get('limit', DEFAULT_LIMIT))
+    if args.get('ips'):
+        search_string = ','.join(argToList(args.get('ips')))
+        search_string = f'?search=address:{search_string}'
+    elif args.get('hostnames'):
+        search_string = ','.join(argToList(args.get('hostnames')))
+        search_string = f'?search=name:{search_string}'
+    elif args.get('asset_id'):
+        search_string = ','.join(argToList(args.get('asset_id')))
+        search_string = f'/{search_string}'
+    elif args.get('search'):
+        search_string = f'?search={args.get("search")}'
+    raw = client.asset_search(search_string)
+    remove_attr = not argToBoolean(args.get('display_attributes', 'False'))
+    remove_svc = not argToBoolean(args.get('display_services', 'False'))
+    message = []
+    if type(raw) is list:
+        raw = raw[:limit]
+        for item_raw in raw:
+            if remove_attr:
+                del item_raw['attributes']
```

Comment:
please use `pop('attributes', None)` as it'll work even when `attributes` is missing.
Same comment for the rest of deletes like this.
```suggestion
                item_raw.pop('attributes', None)
```